### DISCUSSION TITLE
Welcome to Custom-Ore-Generator Discussions!

### DISCUSSION THREAD

**Discussion by DerFrZocker** [DerFrZocker/Custom-Ore-Generator#7] [Announcements]
## 👋 Welcome!
You can use this discussion forum as a place to:

* Ask questions you have about the plugin.
* Get help setting up the plugin.
* Share your creations with others.
If you have a feature request or found a bug in the plugin, please open an issue [here](https://github.com/DerFrZocker/Custom-Ore-Generator/issues).

Response times from the developer can range from a few hours to multiple days. For faster and private support, please consider the paid feature available [here](https://github.com/sponsors/DerFrZocker).

**Comment by Nik2810-Original** [comment]
Moin, ich könnte etwas Hilfe bei deinem Plugin Custom Ore Generator gebrauchen. Irgendwie krieg ich es nicht hin dass er in nicht geladenen Chunks Erze generiert.

**Reply by DerFrZocker** [reply]
Sende deine ore-config, du kannst sie unter `plugins/custom-ore-generator/data/ore-config` finden.
Bitte erstelle in der Zukunft eine eigene Discussion unter [Q&A](https://github.com/DerFrZocker/Custom-Ore-Generator/discussions/categories/q-a).
